src/api/app/assets/stylesheets/webui2/comments.scss

```diff
@@ -3,7 +3,6 @@
     pre {
       @extend .p-2;
       @extend .bg-light;
-      white-space: pre-wrap;
```

Comment:
I think this should be `@extend .text-pre-wrap;` since comments do sometimes contain the tag `pre` (it's not directly in the view, so we cannot attach the class to the tag itself.

Reply:
I changed it now. Thanks for the pointer